lib/ngx/ssl.md

```diff
@@ -299,6 +300,50 @@ This function can be called in whatever contexts where downstream https is used.
 
 [Back to TOC](#table-of-contents)
 
+client_addr
+---------------
+**syntax:** *addr, err = ssl.client_addr()*
+
+**context:** *any*
+
+Returns the client ip address in textual representation of the current SSL connection,
```

Comment:
The API name suggests that it is not specific to the IP protocol. We should support other kinds of clients like unix domain socket clients.